core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-mode.service.spec.ts

```diff
@@ -15,54 +15,52 @@
 /**
  * @fileoverview Unit test for the Translation tab active mode service.
  */
+import { TestBed } from '@angular/core/testing';
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { TranslationTabActiveModeService } from 'pages/exploration-editor-page/translation-tab/services/translation-tab-active-mode.service';
 
-require(
-  'pages/exploration-editor-page/translation-tab/services/' +
-  'translation-tab-active-mode.service.ts');
+describe('Translation tab active mode service', () => {
+  let translationTabActiveModeService : TranslationTabActiveModeService;
 
-describe('Translation tab active mode service', function() {
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
-  var ttams = null;
-
-  beforeEach(angular.mock.inject(function($injector) {
-    ttams = $injector.get('TranslationTabActiveModeService');
-  }));
+  beforeEach(() => {
+    translationTabActiveModeService =
+    TestBed.get(TranslationTabActiveModeService);
```

Comment:
```suggestion
    translationTabActiveModeService = TestBed.get(
      TranslationTabActiveModeService);
```

Break after parenthesis*